lib/file.cpp

```diff
@@ -40,4 +52,21 @@ std::vector<utf8string> file$lines(const std::string &filename)
     return r;
 }
 
+void file$writelines(const std::string &filename, const std::vector<utf8string> &lines)
+{
+    std::string r;
+    std::ofstream f(filename, std::ios::out | std::ios::trunc);
+    if (not f.is_open()) {
+        throw RtlException("FileOpenError", filename);
+    }
+    for (auto s : lines) {
+        if (not s.empty()) {
+            if (not f.write(s.c_str(), s.length())) {
```

Comment:
I did it this way to be able to test the return value, and raise a Neon exception we were not able to write to the file.